### ISSUE TITLE
Images do not have appropriate and descriptive Alt text

### ISSUE THREAD

**Issue by oly-baby** [wagtail/wagtail.org#430]
Images in wagtail about page,who uses wagtail page, etc should have appropriate and descriptive alt text. This is important for users who are blind or have low vision and rely on screen readers.
![wag2](https://github.com/wagtail/wagtail.org/assets/131376909/286c6fa5-4d1e-4e52-a73d-85ed541aa1c3)
![wa3](https://github.com/wagtail/wagtail.org/assets/131376909/3397c749-2034-45dc-9bc6-05fee7c9b87d)
![wa1](https://github.com/wagtail/wagtail.org/assets/131376909/75742e16-4c3e-4ee3-bb97-755d9b344fcb)

**Comment by thibaudcolas** [comment]
@oly-baby 👋 could you share a more actionable report? Only one of the images you share here is on the about page. That image has acceptable alt text as far as I can see. 

If you want to report issues with alt text please take a moment to report what the alt text is for the image that you find has a problem. It’s not super helpful to look at the images’ visuals and have to guess where the images are on the site.
